docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,92 @@
 <Subtitle>Changelogs for each Base UI release.</Subtitle>
 <Meta name="description" content="Changelogs for each Base UI release." />
 
+## v1.0.0-alpha.6
+
+**Feb 6, 2025**
+
+### AlertDialog
+
+- `onOpenChangeComplete` prop (#1305) @atomiks
```

Comment:
The PR numbers have to be converted to markdown links. Authors should be removed.